src/components/Pagination/PaginationContainer/PaginationContainer.js

```diff
@@ -4,10 +4,8 @@ import styled from 'react-emotion';
 import Button from '../../Button';
 
 const Container = styled('div')`
-  ${({ theme, align, justify }) => `
+  ${({ theme }) => `
     display: flex;
-    align-items: ${align};
-    justify-content: ${justify};
```

Comment:
I'd keep some sensible defaults here (I think you used `center` as the default prop for both).

Reply:
Hm, it might be useful to provide a default for the vertical alignment, but I'm not so sure about the horizontal one. I'll add only `align-items`. Perhaps it would be a good idea for the future to add something to the documentation about how one would easily align the component horizontally.